docs/recipes/raster_to_kmz.py

```diff
@@ -0,0 +1,52 @@
+import os
+import zipfile
+
+from affine import Affine
+import numpy as np
+import rasterio
+
+width = 100
+height = 100
+count = 3
+box_affine = Affine(0.01, 0, -36, 0, -0.01, 75)
+crs = {'init': 'epsg:4326'} 
+dtype = "uint8"
+raster_filename = "green_box.tif"
+kml_filename = "green_box.kml"
+kmz_filename = "green_box.kmz"
+kml_im_path = os.path.join("files", os.path.basename(raster_filename))
+
+with rasterio.open(raster_filename, "w", "GTiff", width=width, height=height,
+                   count=count, crs=crs, transform=box_affine,
+                   dtype=dtype) as dest:
+    data = np.zeros([count, height, width], dtype=dtype)
+    data[1, :, :] = 160
```

Comment:
Maybe a comment here about that this is creating a green image / setting the green band to 160